lib/puppet/provider/network_route/routes.rb

```diff
@@ -91,7 +91,8 @@ def self.format_file(_filename, providers)
       raise Puppet::Error, "#{provider.name} is missing the required parameter 'gateway'." if provider.gateway.nil?
       raise Puppet::Error, "#{provider.name} is missing the required parameter 'interface'." if provider.interface.nil?
 
-      contents << "#{provider.network} #{provider.netmask} #{provider.gateway} #{provider.interface} #{provider.options}\n"
+      contents << "#{provider.network} #{provider.netmask} #{provider.gateway} #{provider.interface}"
+      contents << (provider.options == :absent ? "\n" : " #{provider.options}\n")
```

Comment:
I'm not a big fan of encapsulated ternary. What do you think @igalic, do we want to keep it or reformat?